pkg/pfcpsim/session.go

```diff
@@ -0,0 +1,23 @@
+package pfcpsim
+
+import "github.com/wmnsk/go-pfcp/ie"
+
+type PFCPSession struct {
+	LocalSEID uint64
+	PeerSEID  uint64
+
+	PDRs []*ie.IE
+	FARs []*ie.IE
+
+	QERs []*ie.IE
+}
+
+func NewSession() *PFCPSession {
```

Comment:
I'm wondering if we should export `NewSession`. Should a user be able to create a session, without establishing it? I don't think so. 

Also, we should make fields of `PFCPSession` private IMO.

Reply:
made PFCPSession fields private. `NewSession` has been removed, as agreed offline